### ISSUE TITLE
TypeError: toISOString is not a function

### ISSUE THREAD

**Issue by dnikonov** [avocode/avocode#1332]
IMPORTANT: Please try to describe how did it happen.

-----
App v. `2.13.0`
Avocode UID: 24143
```
TypeError: toISOString is not a function
    at String.toJSON (<anonymous>)
    at Object.<anonymous> (/Users/ds-nikodim/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/core-js/modules/_ctx.js:18:15)
    at JSON.stringify (<anonymous>)
    at stringify (/Users/ds-nikodim/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/json-stringify-safe/stringify.js:5:15)
    at Raven._makeRequest (/Users/ds-nikodim/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/raven-js/src/raven.js:1559:22)
    at Raven._sendProcessedPayload (/Users/ds-nikodim/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/raven-js/src/raven.js:1485:56)
    at Raven._send (/Users/ds-nikodim/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/raven-js/src/raven.js:1442:14)
    at Raven._processException (/Users/ds-nikodim/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/raven-js/src/raven.js:1290:14)
    at Raven._handleStackInfo (/Users/ds-nikodim/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/raven-js/src/raven.js:1196:14)
    at Raven.captureException (/Users/ds-nikodim/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/raven-js/src/raven.js:350:18)
    at e.captureException (file:///Users/ds-nikodim/Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:41:10164)
    at file:///Users/ds-nikodim/Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:7:21853
```

**Comment by vojtatranta** [comment]
That's a bug, we will fix it in another minor `2.13.1`

**Comment by vojtatranta** [comment]
At the end, we weren't able to reproduce it, I'll keep it here if similar issue comes

**Comment by vojtatranta** [comment]
We are closer to fix it. 

The fix should land in `2.13.2` however we cannot reliably reproduce it.

**Comment by vojtatranta** [comment]
Actually. We found another source of this bug and it should be fixed in `2.13.3`

FYI: circular JSON of error event sent to sentry
